### PR TITLE
Improve GoReportCard metrics, both gofmt and golint

### PR DESCRIPTION
**Pull request by okartau**
Order of import changed in main.go and network.go,
missing ending newline added in fake.go

[warm-up exercise as just starting with this project. This change is expected to set gofmt metrics to 100%]

### CODE REVIEW

File: mockapihelpers.go [hunk 1]
```diff
@@ -6,6 +6,7 @@ import (
 	api "k8s.io/client-go/pkg/api/v1"
 )
 
+// MockAPIHelpers for testing support
```
Comment (balajismaniam) [inline mockapihelpers.go]: This file is auto-generated. Making manual changes to auto-generated files is not recommended. Please revert this change.

File: mockfeaturesource.go [hunk 2]
```diff
@@ -2,6 +2,7 @@ package main
 
 import "github.com/stretchr/testify/mock"
 
+// MockFeatureSource for testing support
```
Comment (balajismaniam) [inline mockfeaturesource.go]: This file is auto-generated. Making manual changes to auto-generated files is not recommended. Please revert this change.

File: source/fake/fake.go [hunk 3]
```diff
@@ -19,13 +19,15 @@ package fake
 // Source implements FeatureSource.
 type Source struct{}
 
+// Name returns a friendly name for this source.
```
Comment (balajismaniam) [inline source/fake/fake.go]: Maybe change this description to `Name returns a identifier string for this feature source`?

File: source/network/network.go [hunk 4]
```diff
@@ -19,19 +19,20 @@ package network
...
 	"net"
 	"strconv"
 	"strings"
-	"github.com/golang/glog"
 )
 
 // Source implements FeatureSource.
 type Source struct{}
 
+// Name returns a friendly name for this source.
```
Comment (balajismaniam) [inline source/network/network.go]: Maybe change this description to `Name returns a identifier string for this feature source`?

File: source/panic_fake/fake_panic.go [hunk 5]
```diff
@@ -19,8 +19,10 @@ package panic_fake
 // Source implements FeatureSource.
 type Source struct{}
 
+// Name returns a friendly name for this source.
```
Comment (balajismaniam) [inline source/panic_fake/fake_panic.go]: Maybe change this description to `Name returns a identifier string for this feature source`?

File: source/pstate/pstate.go [hunk 6]
```diff
@@ -24,6 +24,7 @@ import (
 // Source implements FeatureSource.
 type Source struct{}
 
+// Name returns a friendly name for this source.
```
Comment (balajismaniam) [inline source/pstate/pstate.go]: Maybe change this description to `Name returns a identifier string for this feature source`?

File: source/rdt/rdt.go [hunk 7]
```diff
@@ -31,9 +31,10 @@ const (
 // Source implements FeatureSource.
 type Source struct{}
 
+// Name returns a friendly name for this source.
```
Comment (balajismaniam) [inline source/rdt/rdt.go]: Maybe change this description to `Name returns a identifier string for this feature source`?

**Comment by okartau** [comment]
I made proposed changes. I used "an identifier" as this seems grammatically more correct.
About generated files, seems no right that generated code drags down quality metrics, is it possible to
affect or improve the genaration part then?

**Comment by balajismaniam** [comment]
Thanks for making these changes. During linting there are ways to ignore files if required.

**Comment by balajismaniam** [comment]
@okartau Any updates on this?

**Comment by okartau** [comment]
rebased and force pushed, should be up to date now

**Comment by marquiz** [comment]
PR #141 is aiming at resolving this
